templates/index.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 
 <body class="main">
-  <a href="https://idp-staging.idmeinc.net/oauth/authorize?client_id=aeab0d4282e2127f900075d9673aeca5&redirect_uri=http://localhost:5000/callback&response_type=code&scope=military" class="idme-btn">
+  <a href="https://api.id.me/oauth/authorize?client_id=aeab0d4282e2127f900075d9673aeca5&redirect_uri=http://localhost:5000/callback&response_type=code&scope=military" class="idme-btn">
```

Comment:
In that case, we should remove it with instructions on how to add their own client credentials